sdk/search/search-documents/src/indexModels.ts

```diff
@@ -37,6 +37,26 @@ export interface SearchIndexingBufferedSenderOptions {
    * Indicates if autoFlush is enabled.
    */
   autoFlush?: boolean;
+  /**
+   * Initial Batch Document Count
+   */
+  initialBatchDocumentCount?: number;
+  /**
+   * Flush Window
```

Comment:
again, maybe "after this number of milliseconds, we will submit the pending batch to the server" or something more descriptive?

Reply:
Added more descriptive comment